pkg/controller/cassandracluster/generator_test.go

```diff
@@ -415,6 +412,27 @@ func TestGenerateCassandraStatefulSet(t *testing.T) {
 	assert.Equal(ccDefault.Spec.BackRestSidecar.Image, api.DefaultBackRestImage)
 }
 
+func TestCassandraStatefulSetHasNoDuplicateVolumes(t *testing.T) {
+	dcName := "dc1"
+	dcRackName := fmt.Sprintf("%s-rack1", dcName)
+
+	_, cc := HelperInitCluster(t, "cassandracluster-2DC.yaml")
+	labels, nodeSelector := k8s.DCRackLabelsAndNodeSelectorForStatefulSet(cc, 0, 0)
+	sts, _ := generateCassandraStatefulSet(cc, &cc.Status, dcName, dcRackName, labels, nodeSelector, nil)
+
+	assert := assert.New(t)
+	cassandraContainer := sts.Spec.Template.Spec.Containers[2]
+	assert.Equal(cassandraContainer.Name, "cassandra")
```

Comment:
```suggestion
	assert.Equal(cassandraContainer.Name, cassandraContainerName)
```

Use of variable name